### PR TITLE
Add isort config

### PR DESCRIPTION
**Pull request by emord**
https://github.com/dimagi/commcare-hq/pull/23799#discussion_r271327474

@proteusvacuum @esoergel

### CODE REVIEW

File: .isort.cfg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[settings]
+multi_line_output=3
+include_trailing_comma=true
```
Comment (emord) [inline .isort.cfg]: I'd also like line_length = 80 because that enforces multiline imports sooner, but decided not to impose too many of my own opinions

**Review by proteusvacuum — APPROVED** [review]

**Comment by esoergel** [comment]
:clap:

File: .isort.cfg (hunk 1 above)
Comment (millerdev) [reply .isort.cfg]: That's the default, isn't it?

**Review by millerdev — COMMENTED** [review]
🎉

File: .isort.cfg (hunk 1 above)
Comment (emord) [reply .isort.cfg]: hmm. default is 79. For some reason I thought that it was picking up HQ's stickler 115 line length. I could just be crazy

**Pull request merged**